.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,95 @@
+<!---
+Step 1 - Title this PR with the following format:
+USWDS - [Package]: [Brief statement describing what this pull request solves]
+eg: "USWDS - Button: Increase font size"
+ -->
 
+## Summary
+_Provide a one or two sentence summary of the update that can be used in the changelog._
+<!--
+A successful summary is written in the past tense and includes:
+1. A benefit statement, and
+2. A description of the update
+See the [USWDS release notes](https://github.com/uswds/uswds/releases) for examples.
 -->
 
-## Description
+### Breaking change
+_Indicate if this update is a breaking change._
+<!--
+Use clear language to:
+1. Indicate if this update is a breaking change.
+2. Explain what actions are required for the user to remediate the break.
+For clarity, include a statement here even if there is no breaking change:
+":warning: This is a breaking change."
+":warning: This is potentially a breaking change."
+"This is not a breaking change."
+-->
+
+## Related issue
+Closes #_[issue_no]_
+<!--
+Every pull request should resolve an open issue.
+If no open issue exists, you can open one here: https://github.com/uswds/uswds/issues/new/choose.
+-->
 
-Include a high-level description of the issue your pull request fixes; include only one issue per pull request. Please also link to the GitHub issue it resolves.
+## Preview link
+Preview link:
+<!-- If available, provide a link to a demo of the solution in action. -->
+
+## Problem statement
+_Summarize the problem this PR solves in a clear and concise statement._
+<!--
+A successful problem statement conveys:
+1. The desired state,
+2. The actual state, and
+3. Consequences of remaining in the current state (who does this affect and to what degree?)
+-->
 
-## Additional information
+## Solution
+_Provide a summary of the solution this PR offers._
+<!--
+It can be helpful if we understand:
+1. What the solution is,
+2. Why this approach was chosen,
+3. How you implemented the change, and
+4. Possible limitations of this approach and alternate solution paths.
+-->
+
+## Major changes
+_For complex PRs, create a list of the significant updates made._
+
+## Testing and review
+_Share recommended methods for reviewing this change._
+<!--
+1. Describe the tests that you ran to verify your changes,
+2. Provide instructions to reproduce these tests, and
+3. Clarify the type of feedback you are looking for at this phase.
+-->
 
-Include any of the following (as necessary): 
+## Dependency updates
+_For PRs that include dependency updates, include a list of the changed dependencies and version numbers._
+| Dependency name              | Previous version | New version |
+| ---------------------------- | :--------------: | :---------: |
+| [Updated dependency example] |     [1.0.0]      |   [1.0.1]   |
+| [New dependency example]     |        --        |   [3.0.1]   |
+| [Removed dependency example] |     [2.10.2]     |     --      |
```

Comment:
Possibly comment this out/remove

Reply:
Commented this out in [b17f97b](https://github.com/uswds/uswds/pull/4842/commits/b17f97b861b592887a7395da2f51ad6b65d44c2f)